cpp/include/rapidsmpf/streaming/core/channel.hpp

```diff
@@ -107,30 +188,9 @@ class Channel {
     }
 
   private:
-    coro::ring_buffer<T, Capacity> rb_;
+    coro::ring_buffer<Message, 1> rb_;
```

Comment:
We should do some experimenting around whether it makes sense to be able to control the buffering capacity of the channel, but for now I think hard-coding to 1 is fine.